tests/testthat/_snaps/trace.md

```diff
@@ -1282,7 +1282,7 @@
     Output
       <error/rlang_error>
       Error in <<CALL `h()`>>:
-      ! `x` must be a single string, not a number.
+      ! `x` must be a single string, not 1.
```

Comment:
Maybe that should be:

> not the number 1.

Reply:
Yeah, I like that.

Reply:
Similar with `not the string "foo"`? Or is that different?